avocado/utils/lv_utils.py

```diff
@@ -170,17 +170,17 @@ def vg_ramdisk_cleanup(ramdisk_filename=None, vg_ramdisk_dir=None,
             errs.append("wipe pv")
             LOGGER.error("Failed to wipe pv from %s: %s", loop_device, result)
 
-        if loop_device in process.system_output("losetup --all"):
-            ramdisk_filename = re.search(r"%s: \[\d+\]:\d+ \(([/\w]+)\)" %
-                                         loop_device,
-                                         process.system_output("losetup --all"))
+        losetup_all = process.run("losetup --all").stdout_text
+        if loop_device in losetup_all:
+            ramdisk_filename = re.search("%s: \[\d+\]:\d+ \(([/\w]+)\)" %
+                                         loop_device, losetup_all)
             if ramdisk_filename is not None:
                 ramdisk_filename = ramdisk_filename.group(1)
 
             for _ in range(10):
                 result = process.run("losetup -d %s" % loop_device,
                                      ignore_status=True, sudo=True)
-                if "resource busy" not in result.stderr:
+                if b"resource busy" not in result.stderr:
```

Comment:
Shouldn't be used `stderr_text` here?

Reply:
It's either text with stderr_text, or bytes with stderr.